### PR TITLE
fix(online-debug): 在线调试体验问题修复

### PR DESCRIPTION
**Pull request by shuzhenyang**
### Description

Fixes # (issue)

### Checklist

- [ ] 填写 PR 描述及相关 issue (write PR description and related issue)
- [ ] 代码风格检查通过 (code style check passed)
- [ ] PR 中包含单元测试 (include unit test)
- [ ] 单元测试通过 (unit test passed)
- [ ] 本地开发联调环境验证通过 (local development environment verification passed)

### CODE REVIEW

**Comment by wklken** [comment]
### Pull Request Summary

#### Title
fix(online-debug): 在线调试体验问题修复

#### Description
This pull request, authored by [shuzhenyang](https://github.com/shuzhenyang), addresses issues related to the online debugging experience.

#### Changes
- **Additions**: 67 lines
- **Deletions**: 11 lines
- **Changed Files**: 5 files

#### Checklist
- PR description and related issue filled
- Code style check passed
- Included unit tests
- Unit tests passed
- Local development environment verification passed

#### Status
- **State**: Open
- **Mergeable**: True (currently in a blocked mergeable state)
- **Reviewers Requested**: [jinquantianxia](https://github.com/jinquantianxia)

You can view more details about this pull request [here](https://github.com/TencentBlueKing/blueking-apigateway/pull/1074).

File: src/dashboard-front/src/views/online-debug/components/edit-table.vue [hunk 1]
```diff
@@ -317,6 +317,13 @@ const addRow = (index: number) => {
 };
 
 const delRow = (index: number) => {
+  if (tableData.value?.length === 1) {
```
Comment (jinquantianxia) [inline src/dashboard-front/src/views/online-debug/components/edit-table.vue:320]: tableData应该至少是个空数组，这个？似乎多余

File: src/dashboard-front/src/views/online-debug/components/request-record.vue [hunk 2]
```diff
@@ -177,6 +179,7 @@ const handleRowClick = async (e: Event, row: Record<string, any>) => {
     await getDetails(row.id, row);
   } else {
     row.isExpand = !row.isExpand;
+    expandIds = expandIds?.filter((id: number) => id !== row.id);
```
Comment (jinquantianxia) [inline src/dashboard-front/src/views/online-debug/components/request-record.vue:182]: 同上

File: src/dashboard-front/src/views/online-debug/components/request-record.vue [hunk 3]
```diff
@@ -215,11 +218,19 @@ const getDetails = async (id: number, row: Record<string, any>) => {
 
   row.editorText = res?.response?.data?.curl;
   row.isExpand = !row.isExpand;
+  expandIds?.push(id);
```
Comment (jinquantianxia) [inline src/dashboard-front/src/views/online-debug/components/request-record.vue:221]: 同上

File: src/dashboard-front/src/views/online-debug/index.vue [hunk 4]
```diff
@@ -277,19 +277,28 @@
           </template>
         </bk-resize-layout>
       </div>
+
+      <div class="exception-part" v-show="!Object.keys(resourceGroup)?.length">
```
Comment (jinquantianxia) [inline src/dashboard-front/src/views/online-debug/index.vue:281]: 这个表达式在template里面多个地方用到，写成computed更好

File: src/dashboard-front/src/views/online-debug/index.vue [hunk 5]
```diff
@@ -844,6 +854,13 @@ const viewDoc = (e: Event) => {
   isShowDoc.value = true;
 };
 
+const openTab = (name?: string) => {
+  if (!name) {
+    return;
+  }
+  window.open(`${globalConfig.PREV_URL}/api-docs/apigw/${common?.apigwName}/${name}`);
```
Comment (jinquantianxia) [inline src/dashboard-front/src/views/online-debug/index.vue:861]: 这个不能用router吗？写死链接，后面不利于维护

**Review by jinquantianxia — APPROVED** [review]

**Pull request merged**